tools/test.py

```diff
@@ -80,14 +80,17 @@ def main(argv):
 
     integration_tests(deno_exe)
 
-    # TODO We currently skip testing the prompt and IsTTY in Windows completely.
+    # TODO We currently skip testing
+    # the prompt, IsTTY and execPath in Windows completely.
     # Windows does not support the pty module used for testing the permission
     # prompt.
     if os.name != 'nt':
         from permission_prompt_test import permission_prompt_test
         from is_tty_test import is_tty_test
+        from exec_path_test import exec_path_test
         permission_prompt_test(deno_exe)
         is_tty_test(deno_exe)
+        exec_path_test(deno_exe)
```

Comment:
Why skip it in Windows?

Reply:
It uses `tty_capture`, which doesn't work there

Reply:
Why? It doesn't need tty_capture.

Reply:
Follow the example here https://github.com/denoland/deno/blob/90c7af27d7959e94a25f635e21b8d77cb347e135/tools/test.py#L29-L38